README.md

```diff
@@ -183,3 +183,6 @@ The tag can be pushed from your local environment, or via the ["Draft a new Rele
 ### Choosing the firmware release to be bundled with the build
 
 The CI automatically downloads a given firmware release from the [ayab-firmware repo](https://github.com/AllYarnsAreBeautiful/ayab-firmware) and packs it into the Desktop release. The name of the firmware release is chosen in [this manifest file](https://github.com/AllYarnsAreBeautiful/ayab-desktop/blob/1.0.0-dev/src/main/resources/base/ayab/firmware/manifest.txt) in the ayab-desktop repo.
+    http://download.qt.io/official_releases/online_installers/qt-unified-mac-x64-online.dmg
```

Comment:
seems that this line does not belong here?

Reply:
ups :3